.rubocop.yml

```
@@ -67,6 +67,12 @@ Style/GuardClause:
 RSpec/ContextWording:
   Enabled: false
 
+RSpec/ExampleLength:
+  Max: 10
+
+RSpec/MultipleExpectations:
+  Enabled: false
```

Não entendi muito bem essa regra, mas acho que seria algo parecido com:
```suggestion
  Max: 10
```

De acordo com:
https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/MultipleExpectations